backend/read_data.py

```diff
@@ -28,12 +35,25 @@ def unbias(display_data: pd.DataFrame):
     Returns:
     - unbiased_data: pd.DataFrame
     """
-    pass
+    new_dataframe=display_data.apply(apply_helper_for_unbias, axis=1)
+    return new_dataframe
+
+
+def apply_helper_for_unbias(row):
+    if row['confusion_value']=='FP':
+        row['confusion_value']='TN'
+    return row
+
 
-def filter_for_valid_transactions():
+
+def filter_for_valid_transactions(display_data: pd.DataFrame):
     """Creates a new dataset without any rows marked as blocked (i.e. False Positive (incorrectly blocked), True Positive (correctly blocked))
     """
-    pass
+    filtered_df = display_data[(display_data['confusion_value'] != 'FP') & (display_data['confusion_value'] != 'TP')]
```

Comment:
Eric specified that we should be returning unconnected copies of the df, if I remember STA130 correctly, changing anything in this returned df would also mutate the original df. I believe there is a df.copy() or something like that that creates an unconnected copy